src/components/ListOfUsers.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import UserDetail from "./UserDetail";
+
+function ListOfUsers(props) {
+  const Users = props.users.map((user, index) => (
+    <UserDetail
+      first_name={user.first_name}
+      last_name={user.last_name}
+      key={index}
+    />
+  ))
+  return (
+    Users[0]
```

Comment:
Here you are only returning the first user... If you want to return all, wrap them in a <div> element.